packages/shared/components/tiles/Tile.svelte

```diff
@@ -13,8 +14,16 @@
         : 'bg-gray-50 dark:bg-gray-900'}
         {classes}
     "
+    class:selected
     on:click
     {...$$restProps}
 >
     <slot />
 </tile>
+
+<style lang="scss">
+    .selected {
+        @apply border border-solid border-blue-500 bg-blue-100;
```

Comment:
having a border only when it's selected will cause the layout to shift, could you try using outline instead??